README.md

```diff
@@ -29,7 +29,8 @@ REST API for managing cache control information for pages within the legacy CMS
 | MONGODB_IS_SSL                     | false                                                      | Switch to use (or not) TLS when connecting to mongodb                                                               |
 | GRACEFUL_SHUTDOWN_TIMEOUT          | 5s                                                         | The graceful shutdown timeout in seconds (`time.Duration` format)                                                   |
 | HEALTHCHECK_INTERVAL               | 30s                                                        | Time between self-healthchecks (`time.Duration` format)                                                             |
-| HEALTHCHECK_CRITICAL_TIMEOUT       | 90s                                                        | Time to wait until an unhealthy dependent propagates its
+| HEALTHCHECK_CRITICAL_TIMEOUT       | 90s                                                        | Time to wait until an unhealthy dependent propagates its                                                            |
```

Comment:
Since you're fixing this line, you may as well fix the description (it looks like it was cropped at some point). It should say:

```
Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
```